README.md

```diff
@@ -1,3 +1,18 @@
 # Futbol
 
-Starter repository for the [Turing School](https://turing.io/) Futbol project.
+**Overview of requirements to consider:**
+
+- Classes should be compact - no longer than 150 lines!
+- Methods should be compact - no longer than 10 lines!
+- Classes should have a single responsibility - must be able to describe what a class 'is' in 1 sentence.
+- Incorporate Modules, Inheritance, and Plain Old Ruby Objects (POROs).
+- While refactoring all existing tests must continue to pass.
+- All methods MUST have an associated test.
+
+### Our design strategy:
+
+- Could give an overview of the classes and why we chose to implement things the way we did (15000' approach)
+- Could speak to the journey that we took (rewriting the code, etc)... what we learned and how it shaped the final product we are turning in.
+- Possibly provide examples of repetitive code we were able to eliminate/consolidate.
+- Where we chose to implement modules, inheritance, etc and why
+- ETC!
```

Comment:
Thank you for updating the README

Reply:
@Callbritton - Yes! Thank you! Making sure we have this deliverable will be so much easier with your brainstorm up!